spec/components/guardian_spec.rb

```diff
@@ -539,15 +550,26 @@
       let(:group_owner) { group_private_topic.user.tap { |u| group.add_owner(u) } }
       fab!(:pm) { Fabricate(:topic) }
 
+      it 'returns true if user trust level matches pm invite site setting' do
+        SiteSetting.min_trust_to_allow_pm_invite = 2
+        expect(Guardian.new(trust_level_2).can_invite_to?(topic)).to be_truthy
```

Comment:
The test here doesn't look right. `topic` is a regular topic and not a PM which we should be testing for.